.github/workflows/k8s-ci.yml

```diff
@@ -150,9 +156,18 @@ jobs:
       - name: Init Hermit
         run: ./bin/hermit env -r >> $GITHUB_ENV
 
+      - name: Kind Config2 Workaround
```

Comment:
Can we have a more informative name here?

Reply:
Step name and comments have been updated.